projects/services-ogc/src/lib/owc/owc-json.service.ts

```diff
@@ -463,13 +550,59 @@ export class OwcJsonService {
         // @TODO
         break;
       case CustomLayertype:
-        // custom layers are meant to be userdefined and not easily encoded in a OWC.
+        // custom layers are meant to be user-defined and not easily encoded in a OWC.
         break;
     }
 
     return rasterLayer$;
   }
 
+  createTmsLayerFromOffering(offering: IOwsOffering, resource: IOwsResource, context: IOwsContext): Observable<CustomLayer> {
+    const layerOptions = this.getLayerOptions(offering, resource, context);
+    const tmsServerUrl = offering.operations.find(o => o.code === 'GetTiles').href;
+    const layer = new CustomLayer({
+      ... layerOptions,
+      type: 'custom',
+      custom_layer: new olVectorTileLayer({
+        declutter: true,
+        source: new olVectorTileSource({
+          attributions: this.getResourceAttribution(resource),
+          format: new olMVT(),
+          tileGrid: createXYZ({
+            minZoom: 0,
+            maxZoom: 12
```

Comment:
Are we fixed here to this zoom steps?

Maybe we can use `resource.properties?.minscaledenominator ?? 0` and `resource.properties.?minscaledenominator ?? 12`

Reply:
I think these should rather be part of the offering, not the resource.properties.
The resource.properties hold for all offerings - so if a resource is offered as a WMS, WFS and TMS, the resource.properties should be valid for all of this. Instead, I'd suggest extending creating a `TmsOffering` type. What do you think?

Reply:
This properties exist on https://docs.opengeospatial.org/is/14-055r2/14-055r2.html#48 which is the Layer itself (`IOwsResource`).

I think each layer should have them because the are on the [`olBaseLayer` ](https://openlayers.org/en/latest/apidoc/module-ol_layer_Base-BaseLayer.html) and on [`ukisLayer`](https://github.com/dlr-eoc/ukis-frontend-libraries/blob/master/projects/services-layers/src/lib/types/Layers.ts#L246) so they are not specified on the offering.

Reply:
Hah, that's already in the standard - that's perfect! Alight, will be added in a bit...

Reply:
This is actually a little complicated, which is why I'd like to double-check my approach with you.
What the standard offers is the scale-denominator, not the zoom. 
We can calculate the zoom based on the projection, like this:

      zoom = log_0.5 (res * 256 / width_projection)

where

      res = scale_denominator * projection_units_per_meter / 2571.42

where
    - 2571.42 is a constant and the number of pixels on 1m screen (using the default assumption of 0.28mm/pixel)
    - `width_projection` is the largest extent of the projection


 (I'm basing this calculation on the [WMS Standard](https://portal.ogc.org/files/?artifact_id=14416), to which the OWC Standard refers for the scale-denominator-field, and the way that [openlayers calculates zoom and resolution](https://openlayers.org/en/latest/doc/tutorials/concepts.html))
The projection we'll use for this is of course the projection mentioned in the OWC file.
Would that be alright with you?

Reply:
Oh, I have overlooked... 
Then we should maybe extent our eoc-owc for `minZoom` and `maxZoom`.

If the document defines a `scaledenominator ` instead then use your approach.
I Think for Web Maps it is easier to use zoom instead of `scaledenominator ` so we should support this as well.

Reply:
Roger that! Will do!

Reply:
done!